### PR TITLE
feat: return Claim from `ContentClaimsLocator#locate`

### PR DESCRIPTION
**Pull request by travis**
Return underlying `Claim`  with `Location` - I made `Claim` mandatory and it didn't seem to break anything, but I'm not sure if there are cases where we might want to return `Location`s without including a `Claim` - open to making this optional!

### CODE REVIEW

**Comment by alanshaw** [comment]
I'd return the claims involved as part of return value of `locate`. A global cache of claims is great but at some point I imagine we'll want to use this not in a worker and limit it's size, but since you've established the convention of accessing any claim from any past call to `locate` you'll potentially break consumers. If you return the claims this library can manage the cache how it likes and the consumer can cache (or not) the claims it is interested in.

**Comment by travis** [comment]
ok great - even easier! updated the PR. Any idea what the test failure might be? it's a very odd one...

**Comment by travis** [comment]
update: figured out the test failure! upgrading the pnpm action fixed it

**Comment by alanshaw** [comment]
Can you add a test/assertion(s)? 🙏

File: src/locator/content-claims.js [hunk 1]
```diff
@@ -82,7 +82,8 @@ export class ContentClaimsLocator {
             site: [{
               location: claim.location.map(l => new URL(l)),
               range: { offset: claim.range.offset, length: claim.range.length }
-            }]
+            }],
+            claim
```
[the remark this answers is not in the document]
Comment (travis) [reply src/locator/content-claims.js:86]: oh jeez good call - will update!

**Comment by travis** [comment]
I'm going to put a pause on this, pending @Peeja's RFC referenced here: https://github.com/storacha/project-tracking/issues/138#issuecomment-2396595316

**Comment by Peeja** [comment]
Pretty sure this is now moot, but we can restore it if we need to.

**Pull request closed without merging**